configuration/pipeline_config.json

```diff
@@ -0,0 +1,98 @@
+{
+  "PipelineName": "SSF-elections",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
+      "ContactsFileName": "ssf_elections_contacts",
+      "ActivationFlowNames": [
+        "ssf_elections_s01e01_activation"
+      ],
+      "SurveyFlowNames": [
+        "imaqal_demog",
+        "imaqal_covid19_demog",
+        "ssf_elections_demog"
+      ],
+      "TestContactUUIDs": [
+        "fad44f22-235a-4821-9780-dfef09076fe1",
+        "3903ad8e-284e-48a0-881b-89c0b06b9334",
+        "45939459-e4d5-4ad6-8650-3e7134b9a035"
+      ]
+    },
+    {
+      "SourceType": "RecoveryCSV",
+      "ActivationFlowURLs": [
+      ]
+    }
+  ],
+  "UuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "IMAQAL",
+    "UuidPrefix": "avf-phone-uuid-"
+  },
+  "OperationsDashboard": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+  },
+  "TimestampRemappings": [
+  ],
+  "SourceKeyRemappings": [
+    {"SourceKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"SourceKey": "Rqa_Elections_S01E01 (Text) - ssf_elections_s01e01_activation", "PipelineKey": "rqa_elections_s01e01_raw", "IsActivationMessage": true},
+    {"SourceKey": "Rqa_Elections_S01E01 (Run ID) - ssf_elections_s01e01_activation", "PipelineKey": "rqa_elections_s01e01_run_id"},
+    {"SourceKey": "Rqa_Elections_S01E01 (Time) - ssf_elections_s01e01_activation", "PipelineKey": "sent_on"},
```

Comment:
do you want to configure the other episodes here yet? (I see that you added them in the other parts of configuration)

Reply:
Yup, updated thanks.